tests/testPupilCoordinates.py

```diff
@@ -1,3 +1,7 @@
+from __future__ import division
+from builtins import zip
+from builtins import range
+from past.utils import old_div
```

Comment:
Okay.  That's weird.  I looked at this PR this morning and saw no old_div occurrences.  Now, however, when I look at the PR, I see everything you're seeing.  Not sure how to explain that.